### ISSUE TITLE
Question about View schema

### ISSUE THREAD

**Issue by kpodp0ra** [teableio/teable#1146]
Quick question, can I safely delete `pluginId` from view schema?

https://github.com/teableio/teable/blob/3bdb38fdb46d0dc6d62f6976d11ff91a6cd9c01f/packages/core/src/models/view/view.schema.ts#L55

I checked code real quick and plugin view always uses `/options.pluginId` (green) property, so I think `/pluginId` (red) isn't used anywhere. 

![](https://github.com/user-attachments/assets/26715542-42d6-498c-a5d1-b2bf54940278)

**Comment by boris-w** [comment]
Yes, this is a property that was inadvertently left in Request Object. Currently, the create view endpoint should not be used to directly create plugin views, as there is a separate dedicated endpoint being used for plugin view creation.
